ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -64,7 +69,9 @@ export class LoggingService {
             body,
             { headers: new HttpHeaders({ 'Content-Type': 'application/json' }) }).pipe(
             catchError((error: any) => {
-                this.logError(this.LOG_SOURCE, `HTTP POST: ${url}. Error:`, error);
+                this.stackdriverErrorReporterService.handleError(`${this.LOG_SOURCE}: HTTP POST: ${url}`);
+                this.stackdriverErrorReporterService.handleError(error);
+                console.error.apply(window.console, error);
```

Comment:
If `LoggingService` has failed to post logs for unknown reasons, fallback to `stackdriver-error-js` for error logging.